File: Even Odd.c
//Author : Rohit Goswami


#include <stdio.h>
int main(int argc, char *argv[])
{
    int i;
    if( argc >= 2 )
    {
       int a=atoi(argv[1]);
        if(a%2==0){ printf("Even");}
        else printf("Odd");
    }
    else
    {
        printf("argument list is empty.\n");
    }
    return 0;
}

File: Even Odd.java
//Author : Rohit Goswami


public class Main
{
	public static void main(String[] args) {
	
	if(args.length>0){
	    
	    int a= Integer.valueOf(args[0]);
	    if(a%2==0){
	        System.out.println("even");
	    }
	    else{ System.out.println("odd");}
	}
	else{
	    System.out.println("not enough arguments");
	}
	
	}
}
